apps/src/music/blockly/blockUtils.js

```diff
@@ -90,7 +90,7 @@ export function installFunctionBlocks(blockMode) {
     ) =>
       simple2FunctionCallGenerator(
         generator.getProcedureName(block.getFieldValue('NAME')),
-        block.id
+        block.getProcedureModel().id
```

Comment:
How does this work exactly? Does the level editor still need to provide a custom ID?

Reply:
I'm also curious about this!  @mikeharv?